src/App.tsx

```diff
@@ -107,47 +108,39 @@ const App: FC = () => {
   }, [dispatch, user]);
 
   useEffect(() => {
```

Comment:
lurer på om dette kanskje kunne vært en egen hook..? 🤔

Reply:
Kan se fordeler med å dra det ut i egen hook, koden blir mer kompakt.

Reply:
ja, det ble lit mer oppdelt og unngår alt for mye kode i `App.jsx`, men her får vi evt en komponent/hook som ikke returnerer noe, som virker litt rart. men kan godt være det er helt ok